dotcom-rendering/src/components/ArticlePage.tsx

```diff
@@ -41,6 +41,44 @@ interface AppProps extends BaseProps {
 	renderingTarget: 'Apps';
 }
 
+/**
+ * @description
+ * returns global styles for article pages
+ */
+const globalStyles = (
+	format: ArticleFormat,
+	darkModeInApps: boolean,
+	renderingTarget: string,
```

Comment:
One last thing, this can be narrowed to `RenderingTarget` which is the union of the `Apps` and `Web` literal strings.

Reply:
done!